CMakeLists.txt

```diff
@@ -118,6 +118,8 @@ if (BUILD_NETRADIANT OR BUILD_Q3MAP2)
 			CMAKE_ARGS -DGAMEPACKS_LICENSE_LIST=none
 			CMAKE_ARGS -DGAMEPACKS_NAME_LIST=${NETRADIANT_GAMEPACKS}
 			CMAKE_ARGS -DBUILD_RADIANT=ON
+		else()
+			CMAKE_ARGS -DBUILD_RADIANT=OFF
 		endif ()
 
 		if (BUILD_Q3MAP2)
```

Comment:
maybe you take the opportunity to also add the else to q3map2

Reply:
I can add it but I don't want to test it, so you need to assure me that it works :) Like q3map2 is an important subcomponent of Radiant so I wouldn't assume it works without verification.